README.md

```diff
@@ -1,2 +1,47 @@
-# opa
-A library of policy algorithms deployed as an agent to policy-enable any product or service
+# OPA: Open Policy Agent
+
+OPA is an open source project which can help policy enable any application.
+
+## What is Policy?
+
+Policy defines expected behavior in response to specific events within an application.
+
+There are many examples of functionality within applications which can benefit from rich policy control, e.g, API authorization, VM and container placement, auto-scaling, auto-healing, etc.
+
+The approach to policy can vary greatly within and across applications. For example, policy is often written in natural language (English), stored in documents such as wiki pages, and enforced manually by the administrators of the the application. 
+
+When the cost of manually maintaining and enforcing policy is too high, policy is often defined in way that computers can understand: the application's implementation is updated to include both the definition of policy and the means for enforcing it. Because most applications are implemented in **imperative languages** (such as Java or Python) the policy definition tends to focus on *how* the policy is enforced rather than *what* the expected behavior or state should be. This makes it hard for people unfamiliar with the application's implementation to know what the expected behavior or state should be.
+
+A better approach is to define policy in a **declarative language** which can be understood by a computer. By using a declarative language, the policy definition can focus on *what* the expected behavior or state should be.
```

Comment:
declarative language decoupled from the system.

Can we also add... Policy can be changed without rebuilding the system.  Multiple systems can all use the same policy because there is just 1 copy.  Policy can be analyzed to understand if there are conflicts [more details].  Overall, pulling policy out of the system into a declarative language makes the systems using that policy easier to build, operate, and maintain.